src/ui/tabs/rental_info_tab.py

```
@@ -6,14 +6,26 @@
 from pathlib import Path # Import Path from pathlib
 import shutil # Import shutil for file operations
 import uuid # Import uuid for generating unique filenames
+import urllib.parse
+import re
+import requests  # Used for downloading remote images
+
+# Suppress SSL certificate warnings when verify=False is used in requests
+try:
+    import urllib3
+except ModuleNotFoundError:
+    import requests.packages.urllib3 as urllib3  # type: ignore
```

Pylint (3.3.7)

[refactor] Use 'from requests.packages import urllib3' instead

(R0402)